build.rs

```diff
@@ -494,6 +494,19 @@ fn build_c_code(
     );
 }
 
+fn cc_builder() -> cc::Build {
+    let mut c = cc::Build::new();
+    let target_os = std::env::var("CARGO_CFG_TARGET_OS").unwrap();
+    if target_os == "wasi" {
+        let wasi_sdk_path =
+            &std::env::var("WASI_SDK_PATH").expect("missing environment variable: WASI_SDK_PATH");
+        c.compiler(format!("{}/bin/clang", wasi_sdk_path).as_str());
+        c.archiver(format!("{}/bin/llvm-ar", wasi_sdk_path).as_str());
+        c.flag(format!("--sysroot={}/share/wasi-sysroot", wasi_sdk_path).as_str());
```

Comment:
I feel like these changes belong in cc-rs. We traditionally avoid doing this kind of discovery. I suggest we remove this change from this PR.